src/olympia/promoted/migrations/0011_create_switch_for_subscription.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.16 on 2020-10-16 14:48
+
+from django.db import migrations
+
+
+def create_waffle_switch(apps, schema_editor):
+    Switch = apps.get_model("waffle", "Switch")
+
+    Switch.objects.create(
+        name="enable-subscription",
```

Comment:
That name isn't very descriptive, maybe `enable-subscriptions-for-promoted-addons` instead ? Anything that indicates what kind of subscription we're talking about.